### PR TITLE
Tox clean up

### PR DESCRIPTION
**Pull request by nabobalis**
Cleaned up tox file comments and removed pytest-logger and the `-s` flag.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -50,7 +50,7 @@ database =
 image =
   # TODO: remove the <0.19 pin when discrepenies in
   # sunpy/image/tests/test_transform.py::test_scale have been resolved
-  scikit-image>=0.16.0,<0.19
+  scikit-image>=0.16.0
```
Comment (nabobalis) [inline setup.cfg:53]: Just to check it works on the CI for the reporting.

File: tox.ini [hunk 2]
```diff
@@ -81,28 +59,20 @@ deps =
...
-
-    # These are specific online extras we use to run the online tests.
+    # These are specific extras we use to run in specific factors
     online: pytest-rerunfailures
     online: pytest-timeout
     online: astroquery
     hypothesis: astroquery
-
-    # We have to pin this to use a wheel for manylinux to compile
-    32bit: cryptography<3.0
```
Comment (nabobalis) [inline tox.ini:92]: We don't build 32bit anymore do we?

File: sunpy/tests/helpers.py [hunk 3]
```diff
@@ -45,9 +44,6 @@
 skip_windows = pytest.mark.skipif(platform.system() == 'Windows', reason="Windows.")
 skip_glymur = pytest.mark.skipif(SKIP_GLYMUR, reason="Glymur can not be imported.")
 skip_ana = pytest.mark.skipif(SKIP_ANA, reason="ANA is not available.")
-skip_32bit = pytest.mark.skipif(SKIP_32, reason="Fails on a 32 bit system.")
```
Comment (nabobalis) [inline sunpy/tests/helpers.py:48]: We don't run anything on 32bit anymore?

**Comment by ConorMacBride** [comment]
I'm just thinking we may need the `-s` to always output the `A summary of the failed tests can be found at: ..../fig_comparison.html`, at least once I sort out https://github.com/matplotlib/pytest-mpl/pull/108. i.e. we want to always compare to baseline images even if the hashes match and the tests pass.

**Comment by Cadair** [comment]
What's making that print? We can probably use pytest to output it with a bit of work?

**Comment by ConorMacBride** [comment]
When we add the "always compare" option to see how the images differ to the baseline, we'll need `-s` (when the figure tests all pass) to see the URL.

Yeah, there is probably a pytest function for showing messages. It's currently just a print().

**Comment by nabobalis** [comment]
To see the url on circleci?

**Comment by ConorMacBride** [comment]
Yes. However, just remove `-s`, as I'll hopefully find a better way to show the URL.

**Comment by ConorMacBride** [comment]
The URL is reported by `pytest-mpl`. By default, it writes its to a random temporary directory.

**Comment by nabobalis** [comment]
At least in our case, circleci reports the URL and we can access it directly from the artefacts page. 
So we can worry about losing that down the road.

**Comment by ConorMacBride** [comment]
That's right! I forgot circleci was also showing the output URL. I was just thinking about `pytest-mpl` more generally. Never mind.

**Comment by nabobalis** [comment]
Yeah, that will need addressing in pytest-mpl.

**Comment by nabobalis** [comment]
I am going to add this to my ever growing list of manual backports.

**Comment by nabobalis** [comment]
This probably doesn't even need backporting.

**Review by wtbarnes — APPROVED** [review]

**Comment by nabobalis** [comment]
There might be a pyinstaller issues with scikit-image but we shall see with the next cronjob.

**Pull request merged**